example/src/App.tsx

```diff
@@ -55,10 +56,18 @@ function Account() {
 function FocusedFlow() {
   return (
     <Stack.Navigator screenOptions={{ title: '' }}>
-      <Stack.Screen name={Routes.PhoneActivation} component={WebView} />
-      <Stack.Screen name={Routes.New} component={WebView} />
-      <Stack.Screen name={Routes.SuccessScreen} component={WebView} />
-      <Stack.Screen name={Routes.Fallback} component={WebView} />
+      <Stack.Screen
+        name={Routes.PhoneActivation}
+        getId={getId}
```

Comment:
Needed as we've removed the `key` from navigation payload. `key` is also removed in React Navigation 8.0.

Would love to explore a different pattern for a custom navigator (eg. `const Stack = createWebStackNavigator(..)`) so that this wouldn't be needed to be set explicitly for all defined screens.